reference.md

```diff
@@ -488,7 +488,7 @@ Example:
 ```
 
 
-Supports destructuring and multiple-value binding.
+Supports destructuring and multiple-value binding, see [`let`](#let-scoped-locals).
```

Comment:
It makes sense to reference `let` from `local` but you can't use `let` where you would use `var`, so I don't think it's needed here.